tests/mypy_test.py

```diff
@@ -22,8 +22,11 @@
 import tomli
 from packaging.requirements import Requirement
 
-from _metadata import PackageDependencies, get_recursive_requirements, read_metadata
-from _utils import (
+sys.path += ["lib"]
```

Comment:
This is pretty unfortunate, and also seems to create a dependency on the working directory from which the code is invoked. Could we instead add `pyproject.toml` to the lib directory to make it installable, then add it as a local dependency to requirements.txt?

Reply:
Another option is to use `Path(__file__).parent.parent / "lib"`. On the other hand, most scripts are already assuming that they're called from the root typeshed directory, as they are using `Path("stubs")` and similar constructs.

Reply:
Personally I'd rather scripts be agnostics of their working directory when possible.